<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserHelpWindow</class>
 <widget class="QDialog" name="UserHelpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Руководство пользователя - RayTracing</string>
  </property>
  <widget class="QPushButton" name="button_prev_page">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/prev-page.png</normaloff>images/prev-page.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QTextBrowser" name="text_help">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>751</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_next_page">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>470</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
    border: 2px solid #094065;
}

QPushButton:pressed {
    background-color:  rgb(118, 118, 118);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>images/next-page.png</normaloff>images/next-page.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QTreeWidget" name="list_of_contents">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>431</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Содержание</string>
    </property>
   </column>
   <item>
    <property name="text">
     <string>Главное меню</string>
    </property>
    <item>
     <property name="text">
      <string>Модуль «Теория»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Демонстрация»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Практика»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Модуль «Тестирование»</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Дополнительно</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Просмотр теоретических материалов</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Демонстрация выполнения алгоритма</string>
    </property>
    <item>
     <property name="text">
      <string>Настройки демонстрации</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пошаговое выполнение</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Практические задания</string>
    </property>
    <item>
     <property name="text">
      <string>Получение лабораторных работ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сдача выполненных работ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Доработка и корректировка работы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Статистика по выполненным работам</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Тестирование</string>
    </property>
    <item>
     <property name="text">
      <string>Первоначальные настройки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Режимы тестирования</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Прохождение тестов</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Статистика по пройденным тестам</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text">
     <string>Руководство пользователя</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>О программе</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
